zebra-consensus/src/block/tests.rs

```diff
@@ -10,9 +10,74 @@ use tower::{util::ServiceExt, Service};
 
 use zebra_chain::block::Block;
 use zebra_chain::block::BlockHeader;
-use zebra_chain::serialization::ZcashDeserialize;
+use zebra_chain::serialization::{ZcashDeserialize, ZcashDeserializeInto};
 use zebra_chain::transaction::Transaction;
 
+mod utils {
```

Comment:
if this is only going to be used in `zebra-consensus` we might want to move these utils into `zebra-test`.

Reply:
I think that could be a good idea however we are going to need a lot of dependencies into the `zebra-test` including but not limited to `zebra-chain`, `zebra-state`, `tower`, `spandoc`, etc.

Reply:
I think that will be fine. The `zebra-state` dependency might be an issue but I think as long as we don't use any of the stuff that depends on `zebra-state` in `zebra-state` itself we won't run into the multiple dependency version issue we ran into with `zebra-test` and `zebra-chain`.

We might even want to setup a `zebra_test::prelude` module to make it easier to bring in commonly used items for tests.

Reply:
Moved in https://github.com/ZcashFoundation/zebra/pull/675/commits/0b8468b12ef61fa6f78a2d85f5ae39e6fa49fd07 and https://github.com/ZcashFoundation/zebra/pull/675/commits/a2103aecb689be278f47d781aaeed86bb81b27af